assets/blocks/progress-bar.js

```diff
@@ -0,0 +1,14 @@
+import { render } from 'react-dom';
```

Comment:
Dropping a note here to mention that we don't typically use React directly in WordPress. Instead, we use the [`@wordpress/element` package](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-element/), which is an extra layer that sits on top of React.

You can reference the [setup wizard](https://github.com/Automattic/sensei/blob/master/assets/setup-wizard/index.js) or the [import functionality](https://github.com/Automattic/sensei/blob/master/assets/data-port/import.js) as examples.

Reply:
Oh i did miss it, thanks, I will close this PR since I had to do some reading with session one